exporters/zipkin/src/main/java/io/opentelemetry/exporter/zipkin/ZipkinSpanExporterBuilder.java

```diff
@@ -186,6 +190,16 @@ public ZipkinSpanExporterBuilder setMeterProvider(MeterProvider meterProvider) {
     return this;
   }
 
+  /**
+   * Sets the {@link InternalTelemetrySchemaVersion} defining which self-monitoring metrics this
+   * exporter collects.
+   */
+  public ZipkinSpanExporterBuilder setInternalTelemetry(InternalTelemetrySchemaVersion level) {
```

Comment:
#nit: naming. its not obvious what a method named `setInternalTelemetry` would do. Consider: 
```suggestion
  public ZipkinSpanExporterBuilder setInternalTelemetrySchemaVersion(InternalTelemetrySchemaVersion level) {
```

Or since that's verbose, perhaps we rename `InternalTelemetrySchemaVersion` -> `InternalTelemetryVersion` and rename this method to `setInternalTelemetryVersion`?

(This advice applies to all the builders)